### ISSUE TITLE
Wrong Syntax in Week1 Readme File

### ISSUE THREAD

**Issue by ammarvehbi** [HackYourFuture/databases#227]
Some of the apostrophes have to be changed into single quotes to run the code properly. The parts that have to be corrected in "setting up code" must be as shown in bullets:

- create user 'hyfuser'@'localhost' identified with mysql_native_password by 'hyfpassword';

```
mysql> create user ‘hyfuser’@‘localhost’ identified with mysql_native_password by ‘hyfpassword’;
# This command creates a user 'hyfuser' with password 'hyfpassword' for
# the database server at 'localhost'
```

- grant all privileges on *.* to 'hyfuser'@'localhost';

```
mysql> grant all privileges on *.* to ‘hyfuser’@‘localhost’;
# This command gives all permissions to user 'hyfuser'.
(*.*) means every table of every database.
```

**Comment by unmeshvrije** [comment]
this has been resolved.

**Comment by wilgert** [comment]
Closed via: #229
